packages/camera/camera/lib/camera.dart

```diff
@@ -433,40 +316,18 @@ class CameraController extends ValueNotifier<CameraValue> {
   /// Throws a [CameraException] if image streaming or video recording has
   /// already started.
   // TODO(bmparr): Add settings for resolution and fps.
-  Future<void> startImageStream(onLatestImageAvailable onAvailable) async {
+  Widget buildView() {
```

Comment:
Rename to `buildCameraPreview`
```suggestion
  Widget buildCameraPreview() {
```